package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "4.3.22",
+  "version": "4.4.0",
```

Comment:
Since SPs/GPs are new, I thought this would be a good time to update the minor version number.